scripts/check_lb_data_files.sh

```diff
@@ -23,7 +24,20 @@ do
     run_schema_validator "$f"
 done
 
+find "${path_to_vt_src_dir}/examples" -iname "*.json" | while read f
```

Comment:
`find` can take multiple directories as arguments as in `find /usr/bin/ /bin/ /usr/local/bin/ -name "*php*"`.

Reply:
Multiple search strings too. Something like this could (maybe?) work:
```suggestion
find . "${path_to_vt_src_dir}/examples" -iname "*.json" -o -iname "*.json.br" | grep -v "compile_commands" | while read f
```